QStore4S/src/main/java/edu/asu/qstore4s/repository/IElementNeo4jConnector.java

```diff
@@ -0,0 +1,12 @@
+package edu.asu.qstore4s.repository;
+
+/**
+ * 
+ * @author Rachita Satyasi Connector to QStore to get the total number of nodes
```

Comment:
your description of the class should go above the author

also, the connector connects to Neo4j not QStore. And maybe make it more general, there might be more methods added to this class.  Say something that makes it clear that this class provides method for getting information about Elements in Neo4j (you can create a link to the element class in your java doc. start typing an at-sign and select link I believe).